src/views/AgmExtension.vue

```diff
@@ -72,6 +72,31 @@
               :evaluateResult="extensionRequestValid"
             />
 
+            <!-- Folio Number -->
+            <section
+              v-if="!IsAuthorized(AuthorizedActions.STAFF_PAYMENT)"
+            >
+              <header>
+                <h2 id="folio-number-header">
+                  Folio or Reference Number (Optional)
+                </h2>
+                <p>
+                  This is meant for your own tracking purposes and will appear on your receipt.
+                </p>
+              </header>
+              <div
+                id="folio-number-section"
+                :class="{ 'invalid-section': !folioNumberValid && showErrors }"
+              >
+                <TransactionalFolioNumber
+                  :accountFolioNumber="getFolioNumber"
+                  :transactionalFolioNumber="getTransactionalFolioNumber"
+                  @update:transactionalFolioNumber="onTransactionalFolioNumberChange"
```

Comment:
I think `update` is supposed to be used with a prop to accomplish v-model functionality (ie, the same variable is synced to and from the sub-component).

But you don't have a single prop for FN to do this with.

I suggest using a "change" event instead of update.